src/Components/Modal.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react' 
+import './components.css';
+import CloseButton from './CloseButton';
+import CloseParagraph from './CloseParagraph';
+
+/**
+ * This is a component that return a modal interface
+ */
+
+function Modal({setModalOpen}){
+    const modalBody = `Lorem ipsum dolor sit amet,consectetur adipiscing elit,
+sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+ Felis imperdiet proin fermentum leo vel orci porta non pulvinar.`;
+    const modalTitle = `Esto es un modal`;
```

Comment:
Same here. This should be a prop.

Reply:
Same here, you can see my new solution in my last commit.